bcda/auth/middleware.go

```diff
@@ -108,17 +108,17 @@ func handleTokenVerificationError(ctx context.Context, w http.ResponseWriter, rw
 
 		switch err.(type) {
 		case *customErrors.ExpiredTokenError:
-			rw.Exception(ctx, w, http.StatusUnauthorized, responseutils.ExpiredErr, "")
+			rw.Exception(ctx, w, http.StatusUnauthorized, http.StatusText(http.StatusUnauthorized), responseutils.ExpiredErr)
 		case *customErrors.EntityNotFoundError:
-			rw.Exception(ctx, w, http.StatusForbidden, responseutils.UnauthorizedErr, responseutils.UnknownEntityErr)
+			rw.Exception(ctx, w, http.StatusForbidden, http.StatusText(http.StatusForbidden), responseutils.UnauthorizedErr)
 		case *customErrors.RequestorDataError:
-			rw.Exception(ctx, w, http.StatusBadRequest, responseutils.InternalErr, "")
+			rw.Exception(ctx, w, http.StatusBadRequest, http.StatusText(http.StatusBadRequest), responseutils.InternalErr)
```

Comment:
Perhaps RequestErr instead of InternalErr?

Reply:
I updated some of the mismatched HTTP statuses and the response util strings while doing a larger set of changes that I decided to move into https://jira.cms.gov/browse/BCDA-7962 and I reverted some changes unintentionally, with this being one of them - thanks for catching!